contracts/multisig/src/contract.rs

```diff
@@ -588,12 +586,12 @@ mod tests {
             assert!(!res
                 .events
                 .iter()
-                .any(|e| e.ty == "signing_completed".to_string())); // event is not re-emitted during grace period
+                .any(|e| e.ty == "signing_completed".to_string())); // event is not re-emitted before expiry
```

Comment:
```suggestion
                .any(|e| e.ty == "signing_completed".to_string())); // event is not re-emitted
```
This is basically just testing the event is emitted once.